CHANGELOG.md

```diff
@@ -0,0 +1,17 @@
+# Changelog
```

Comment:
A proposal to implement a file for Changelog with changes that are not yet in any version?

I find it easier to send our updated changelog. What do you think?

@eduardojmatos @yvessacurae @caioluanc @evemontalvao @gisantin @pyymenta

Reply:
I believe it will be well documentated if we structure our changelog, into [release sections](https://github.com/getninjas/niten-tokens/releases). What do you think?

Reply:
@pyymenta So this release gets kind of hard to document when a release with multiple changes comes in.

The intention of CHANGELOG.md is that it is always up to date with things coming into the master and is not necessarily in a release or does not need to create a release.

Reply:
Yes, now I get it. Good idea! :rocket: